app/templates/1_jobs.pug

```diff
@@ -1,4 +1,4 @@
-section#firstPage.special.section(data-scrollax-parent='true')
+section#firstPage.special.section(data-scrollax-parent='true', style='display:none')
```

Comment:
Then you can remove all hidden classes like this `$('.fullpage-wrapper > section.hidden').removeClass('hidden')`